src/Sylius/Bundle/ResourceBundle/Controller/ResourcesResolver.php

```diff
@@ -30,11 +30,16 @@ public function getResources(RequestConfiguration $requestConfiguration, Reposit
             return $resources;
         }
 
-        if (!$requestConfiguration->isPaginated() && !$requestConfiguration->isLimited()) {
+        if (
+            ! $requestConfiguration->isPaginated() &&
```

Comment:
I thought the same.
Perhaps something like:
$this->hasFilterRequirements($requestConfiguration)
or $this->isDefaultResourceRequest($requestConfiguration)

ping @lchrusciel 
please let me know if you have some idea or if one of these names fit for this purpose